src/server.py

```diff
@@ -448,7 +488,13 @@ def play_sound(self, sound_event_id, connection, sender, path, iface,
             self.players[uuid_].connect("error", self.__player_error_cb,
                                         sound_event_id, uuid_,
                                         connection, path, iface)
+            self._apply_on_loops(uuid_, 'pause')
             self.players[uuid_].play()
+            # if a priority loop already playing, stay paused until stops
+            if (self.players[uuid_].loop and
+                    not self.players[uuid_].priority and
+                    self._priority_playing()):
+                self.players[uuid_].pause()
```

Comment:
```
else:
     self.players[uuid_].play()
```

Otherwise if it have to be paused it would pass unnecessarily through the PLAYING state.